profile_bluesky/startup/instrument/devices/monochromator.py

```diff
@@ -26,8 +26,11 @@ class Monochromator(KohzuSeqCtl_Monochromator):
 
     table_x = Component(EpicsMotor,'m7',labels=('motor','monochromator'))
     table_y = Component(EpicsMotor,'m8',labels=('motor','monochromator'))
-
+    
+    energy = Component(EpicsSignal, "BraggERdbkAO", write_pv="BraggEAO",
```

Comment:
Normally, add the `put_complete=True` keyword argument here rather than writing the internal structure as below.